cmd/api-linter/github_actions.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/googleapis/api-linter/lint"
+)
+
+// printGithubActions returns lint errors in GitHub actions format.
+func printGithubActions(responses []lint.Response) ([]byte, error) {
+	var buf bytes.Buffer
+	for _, response := range responses {
+		for _, problem := range response.Problems {
+			// lint example:
+			// ::error file={name},line={line},endLine={endLine},title={title}::{message}
+			// https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message
+
+			fmt.Fprintf(&buf, "::error file=%s", response.FilePath)
+			if problem.Location != nil {
+				fmt.Fprintf(&buf, " line=%d", problem.Location.Span[0])
+				fmt.Fprintf(&buf, " col=%d", problem.Location.Span[1])
+				fmt.Fprintf(&buf, " endLine=%d", problem.Location.Span[2])
+				fmt.Fprintf(&buf, " endColumn=%d", problem.Location.Span[3])
+			}
+
+			runeThatLooksLikeTwoColonsButIsActuallyTwoArmenianFullStops := "։։"
```

Comment:
haha I got a kick out of this one. Can you add a comment to this describing why this is necessary (b.c GH uses the `::` as a delimiter already I'm guessing) and perhaps rename this variable to "ruleDelimiter" instead?